FreeRTOS/FreeRTOS-Partner-Supported-Demos
# cortex-m85: Add Task Dedicated PAC Key example

Description
-----------

This example demonstrates the soundness of the procedure used to stack/unstack the tasks' dedicated PAC key. The example is based on Corstone-315 Ecosystem Fixed Virtual Platform (Arm Cortex-M85 CPU and Ethos-U65 NPU).

The example consists of two main tasks (`prvFirstTask` and `prvSecondTask`). These tasks' job is to make sure that the task's randomly generated PAC keys are stacked/unstacked robustly without being altered.

Test Steps
-----------


Related Issue
-----------


Depends on FreeRTOS-Kernel PR: https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/1195


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
